apps/system/vi/Makefile

```diff
@@ -64,7 +64,7 @@ APPNAME = vi
 FUNCNAME = vi_main
 PRIORITY = SCHED_PRIORITY_DEFAULT
 STACKSIZE = 16384
```

Comment:
Because you change the type to sync, vi application will be executed on TASH task. TASH has 4KB stack but because it uses stack internally, vi could use few stack size. We need to check TASH stack size.

Reply:
How about reverting the THREADEXEC type as async and setting the priority as higher than TASH? In this case, we can vi with its own stack.

Reply:
I'll try to test both.

Reply:
@sunghan-chang @jeongchanKim
Tested with ASYNC mode and higher priority. But the input is ignored by every 2 steps.
So I removed the vi's own stack size to be part of the Tash task.